app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -386,7 +386,7 @@ def center_toolbar_filename_classic
     to_display = %w[availability_zones cloud_networks cloud_object_store_containers cloud_subnets configured_systems
                     cloud_tenants cloud_volumes ems_clusters flavors floating_ips host_aggregates hosts
                     network_ports network_routers network_services orchestration_stacks resource_pools
-                    security_groups security_policies security_policy_rules storages]
+                    security_groups security_policies security_policy_rules storages physical_storages]
```

Comment:
instead of adding it here and https://github.com/ManageIQ/manageiq-ui-classic/pull/7407/files#diff-e0698c70efe7e6235686a03af47a4660R484, you can add `toolbar :physical_storage, :physical_storages` in PhysicalStorage controller [example](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/chargeback_rate_controller.rb#L375)